### DISCUSSION TITLE
Renderless attribute

### DISCUSSION THREAD

**Discussion by N1ebieski** [filamentphp/filament#14198] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

Is it possible to give the livewire attribute [Renderless](https://livewire.laravel.com/docs/actions#skipping-re-renders) to specific actions?

For example, if we have a datatable with the EditAction that opens modal, it is not neccessary to refresh the whole datatable component each time.

**Comment by e2d2-dev** [comment]
i'm pretty sure you watched a youtube video @laravel channel. your question is not precise enough, because the whole presentation was about avoid doing mistakes. sorry for being rough, but a clean talk is way easier than talking around stuff.

**Reply by N1ebieski** [reply]
Yes i watched [Livewire Beyond the Basics](https://www.youtube.com/watch?v=857ean0JIHE) but i don't understand why you're rough. I just asked the question whether it is possible in Filament to use the Renderless attribute...

In my other project built without using Filament, I added Renderless to my modals' actions and it works great. The response time is much faster.

**Comment by e2d2-dev** [comment]
for me it's pretty fast. most js stuff only is way slower than laravel with filament

**Comment by indigoram89** [comment]
+1
